client/landing/stepper/declarative-flow/internals/steps-repository/site-migration-instructions/site-preview/hooks/use-site-preview-mshot-image-handler.ts

```diff
@@ -89,7 +89,7 @@ export const useSitePreviewMShotImageHandler = ( url: string = '' ) => {
 	}, [ previewRef ] );
 
 	const createScreenshots = ( url: string ) => {
-		const isEnabled = config.isEnabled( 'migration-flow/new-migration-instructions-step' );
+		const isEnabled = getLocaleSlug()?.startsWith( 'en' ); // Previous 'migration-flow/new-migration-instructions-step'.
```

Comment:
I left this mark with _"migration-flow/new-migration-instructions-step"_, so it's easier to Search for the occurrences that we still need to remove later.